server/src/ai/ai.functions.ts

```diff
@@ -0,0 +1,53 @@
+import dotenv from 'dotenv';
+import OpenAI from "openai";
+import { Reviews } from "../../db/schema";
+
+dotenv.config();
+const openai = new OpenAI({
+  apiKey: process.env.OPENAI_API_KEY,
+});
+
+/** Docstring for makeSummary. Takes in all reviews from a course as text and 
+ * creates a 50 word summary of those reviews.
+ * @params all reviews from a course
+ * @returns summary of reviews
+ */
+async function makeSummary(text: string) {
```

Comment:
A question I have here is does it make a difference for ChatGPT to read a whole block of text, where reviews are strung one after the other, compared to us telling them this is a list of individual reviews?